overlord/patch/patch.go

```diff
@@ -112,68 +108,28 @@ func maybeResetSublevelForLevel60(s *state.State, sublevel *int) error {
 		return err
 	}
 
-	raw, ok := snaps["core"]
+	_, ok := snaps["core"]
```

Comment:
this bit also doesn't make sense, there might be no core around at all in a snapd only world

Reply:
You're right of course, fixed, thank you!